services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkPageHeader.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Icon, Tag } from "antd";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import * as router from "@/constants/routes";
+import CustomPropTypes from "@/customPropTypes";
+import * as Strings from "@/constants/strings";
+
+const propTypes = {
+  noticeOfWork: CustomPropTypes.nowApplication.isRequired,
+  inspectorsHash: PropTypes.objectOf(PropTypes.string).isRequired,
+  noticeOfWorkPageFromRoute: CustomPropTypes.noticeOfWorkPageFromRoute.isRequired,
+};
+
+const NoticeOfWorkPageHeader = (props) => {
+  const getNumber = () => {
```

Comment:
Is there value to these three functions being functions? "props.noticeOfWork.now_number || Strings.EMPTY_FIELD" needs to be evaluated on every call instead of just once if it is a function vs constant assignment.

Reply:
Good catch, derp moment on my part!